CenterForOpenScience/osf.io
# [OSF Institutions] Update CSIC; Add ASU and SC [SVCS-452][SVCS-657][SVCS-658]

## Purpose

Add two new institutions and enable login for University of South Carolina Libraries (SC) and Arizona State University (ASU).

Update one existing institution Spanish National Research Council (CSIC). No OSF side change, CAS server-side settings update only.

## Changes

### SC

* Project Page
![sc-project-page](https://user-images.githubusercontent.com/3750414/37216451-c6bb8d8c-2388-11e8-8244-5211158d9187.png)

* Institution Page
![sc-institution-page](https://user-images.githubusercontent.com/3750414/37216457-cc82557a-2388-11e8-9ad5-17d218febf93.png)

* Login Page
![sc-login-page](https://user-images.githubusercontent.com/3750414/37216464-d023119c-2388-11e8-8417-a7d558be2d3a.png)

### ASU

* Project Page
![asu-project-page](https://user-images.githubusercontent.com/3750414/37216474-d5d49ea8-2388-11e8-9a01-0e71d18ab536.png)

* Institution Page
![asu-institution-page](https://user-images.githubusercontent.com/3750414/37216480-dbd9f3b6-2388-11e8-9ad4-dc12dfb81c38.png)

* Login Page
![asu-login-page](https://user-images.githubusercontent.com/3750414/37216495-e5b1c012-2388-11e8-9f5c-799564bc1ab5.png)

## QA Notes

* This is a hotfix, only available on Prod and Test
* Go to the institution pages and see if it looks good:
  * CSIC: No QA, server settings update only.
  * SC: https://osf.io/institutions/sc/
  * ASU: https://osf.io/institutions/asu/

## Deployment Notes

- [x] Blocked by CAS server-side private settings @icereval 
  - Please refer to the tickets for the configuration
- [x] Blocked by external domain certificate @mattclark 
- [ ] Requires institution population for Prod and Test @sloria 

## Side Effects

No

## Ticket

https://openscience.atlassian.net/browse/SVCS-452
https://openscience.atlassian.net/browse/SVCS-657
https://openscience.atlassian.net/browse/SVCS-658
